_includes/plugins/katello/sidebar_nightly.html

```diff
@@ -162,19 +162,32 @@
 
 <ul class="header nav nav-list bs-docs-sidenav">
   <li class="item">
-    <a href="/plugins/katello/nightly/troubleshooting/index.html">6. Troubleshooting</a>
+    <a>6. Advanced</a>
+  </li>
+
+  <ul class="sub-header-static nav nav-list bs-docs-sidenav">
+    <li class="item">
+      <a href="/plugins/katello/{{page.version}}/advanced/certificates.html">
+        6.1 Certificates
+      </a>
+    </li>
+</ul>
+
+<ul class="header nav nav-list bs-docs-sidenav">
+  <li class="item">
+    <a href="/plugins/katello/{{page.version}}/troubleshooting/index.html">7. Troubleshooting</a>
   </li>
 </ul>
 
 <ul class="header nav nav-list bs-docs-sidenav">
   <li class="item">
-    <a href="/plugins/katello/nightly/api/index.html">7. Api Documentation</a>
+    <a href="/plugins/katello/{{page.version}}/api/index.html">8. API Documentation</a>
```

Comment:
Should we fix all the nightly links at the same time?

Reply:
If https://github.com/theforeman/theforeman.org/pull/1288 gets merged before this it'll work out nicely because I did all of them there